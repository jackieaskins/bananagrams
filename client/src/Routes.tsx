import React from 'react';
import { Route, Switch } from 'react-router-dom';

import Game from './games/Game';
import Home from './Home';
import NotFound from './NotFound';

type RoutesProps = {};

const Routes: React.FC<RoutesProps> = () => (
  <Switch>
    <Route path="/" exact component={Home} />
    <Route path="/game" exact component={Game} />
    <Route component={NotFound} />
  </Switch>
);

export default Routes;
