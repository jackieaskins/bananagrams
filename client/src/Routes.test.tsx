import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Routes from './Routes';

describe('Routes', () => {
  const renderer = ShallowRenderer.createRenderer();

  test('renders properly', () => {
    expect(renderer.render(<Routes />)).toMatchSnapshot();
  });
});
