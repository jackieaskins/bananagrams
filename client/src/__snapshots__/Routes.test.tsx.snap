// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes renders properly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/game"
  />
  <Route
    component={[Function]}
  />
</Switch>
`;
